Sources/StreamChat/Controllers/ChannelController/ChannelController+SwiftUI_Tests.swift

```diff
@@ -158,16 +158,19 @@ extension _ChatMessage {
             quotedMessageId: nil,
             isSilent: false,
             reactionScores: ["": 1],
-            author: .init(id: .unique),
-            mentionedUsers: [],
             threadParticipants: [],
-            attachments: [],
-            latestReplies: [],
             localState: nil,
             isFlaggedByCurrentUser: false,
-            latestReactions: [],
-            currentUserReactions: [],
             isSentByCurrentUser: false
         )
+        
+        message.$_author = { .init(id: .unique) }
+        message.$_mentionedUsers = { [] }
+        message.$_attachments = { [] }
+        message.$_latestReplies = { [] }
+        message.$_latestReactions = { [] }
+        message.$_currentUserReactions = { [] }
+        
+        return message
```

Comment:
Do I understand it correctly by introducing the `@Cahced` wrappers we lost the compiler support for those values? That's a HUGE issue. The initializer was generated, right? We need to fix this and write the initializer manually. It can't stay like this.